crates/cargo-gpu/src/args.rs

```diff
@@ -0,0 +1,97 @@
+//! Args for building and installing.
+
+use spirv_builder::SpirvBuilder;
+
+/// All args for a build and install
+#[derive(clap::Parser, Debug, Clone, serde::Deserialize, serde::Serialize)]
+pub struct AllArgs {
+    /// build args
+    #[clap(flatten)]
+    pub build: BuildArgs,
+
+    /// install args
+    #[clap(flatten)]
+    pub install: InstallArgs,
+}
+
+/// Args for just a build
+#[derive(clap::Parser, Debug, Clone, serde::Deserialize, serde::Serialize)]
+pub struct BuildArgs {
+    /// Path to the output directory for the compiled shaders.
+    #[clap(long, short, default_value = "./")]
+    pub output_dir: std::path::PathBuf,
+
+    /// Watch the shader crate directory and automatically recompile on changes.
+    #[clap(long, short, action)]
+    pub watch: bool,
+
+    /// the flattened [`SpirvBuilder`]
+    #[clap(flatten)]
+    #[serde(flatten)]
+    pub spirv_builder: SpirvBuilder,
+
+    ///Renames the manifest.json file to the given name
+    #[clap(long, short, default_value = "manifest.json")]
+    pub manifest_file: String,
+}
+
+/// Args for an install
+#[derive(clap::Parser, Debug, Clone, serde::Deserialize, serde::Serialize)]
+pub struct InstallArgs {
+    /// path to the `rustc_codegen_spirv` dylib
+    #[clap(long, hide(true), default_value = "INTERNALLY_SET")]
+    pub dylib_path: std::path::PathBuf,
+
+    /// Directory containing the shader crate to compile.
+    #[clap(long, default_value = "./")]
+    pub shader_crate: std::path::PathBuf,
+
+    #[expect(
+        clippy::doc_markdown,
+        reason = "The URL should appear literally like this. But Clippy wants it to be a in markdown clickable link"
```

Comment:
Just a little typo here. 

"But Clippy wants it to be a markdown..."

Reply:
done in 2 places